pkg/storage/s3/s3_test.go

```diff
@@ -33,14 +32,37 @@ func (s *Storage) clear() error {
 			Key:    o.Key,
 		}
 
-		_, err := s.s3API.DeleteObjectWithContext(ctx, delParams)
+		_, err := s.s3API.DeleteObject(delParams)
 		if err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
+func (s *Storage) createBucket() error {
+	if _, err := s.s3API.CreateBucket(&s3.CreateBucketInput{Bucket: aws.String(s.bucket)}); err != nil {
+		if aerr, ok := err.(awserr.Error); ok {
+			switch aerr.Code() {
+			case s3.ErrCodeBucketAlreadyOwnedByYou:
+				return nil
+			case s3.ErrCodeBucketAlreadyExists:
+				return nil
+			default:
+				return aerr
+			}
+		} else {
+			return err
+		}
+	}
+
+	if err := s.s3API.WaitUntilBucketExists(&s3.HeadBucketInput{Bucket: aws.String(s.bucket)}); err != nil {
```

Comment:
What should be our timeout time? Now by default it will fail after 100sec. https://github.com/aws/aws-sdk-go/blob/3ca7919959ff8f930d28bc50e06d0c229177a291/service/s3/waiters.go#L93

Reply:
there is a config package with S3Config which includes Timeout property
use it like this `s.s3Conf.TimeoutDuration()`